anvil/build.rs

```diff
@@ -6,4 +6,7 @@ fn main() {
     *config.git_mut().sha_kind_mut() = ShaKind::Short;
     vergen::vergen(config)
         .unwrap_or_else(|e| panic!("vergen crate failed to generate version information! {}", e));
+
+    // Supports new Apple Silicon packages installed by Homebrew
+    println!("cargo:rustc-link-search=/opt/homebrew/Cellar/libusb/1.0.26/lib");
```

Comment:
I *think* Homebrew should link these libraries for you in `/usr/local/lib` - can you try running `homebrew link <name of libusb package>`?

Reply:
Yes, I was hoping this would fix it as well but it doesn't (at least not for me -- maybe I messed something up).

```
➜  Documents brew link libusb
Warning: Already linked: /opt/homebrew/Cellar/libusb/1.0.26
To relink, run:
  brew unlink libusb && brew link libusb
➜  Documents brew unlink libusb && brew link libusb                           
Unlinking /opt/homebrew/Cellar/libusb/1.0.26... 6 symlinks removed.
Linking /opt/homebrew/Cellar/libusb/1.0.26... 6 symlinks created.
➜  Documents ls /usr/local/Frameworks/
Frameworks/  Homebrew/    bin/         etc/         include/     lib/         libexec/     n/           opt/         sbin/        share/       var/  
```

Can always just manually link, but that doesn't seem like a good long term solution.

Reply:
Sorry, to be clear in the above comment, the path we're looking for is `/usr/local/Cellar/...`

Reply:
That's strange. cc @gakonst you use M1 too I think, how did you get it to work?